### ISSUE TITLE
Please consider LinXgboost

### ISSUE THREAD

**Issue by farjami** [cerlymarco/linear-tree#4]
dear Marco 
please consider LinXgboost compared to your linear-tree
https://github.com/ldv1/LinXGBoost
https://arxiv.org/abs/1710.03634
thank you in advance

**Comment by cerlymarco** [comment]
Hi, 

LinXgboost is a gradient-boosting algo. Maybe today it's a little bit outdated since the standard XGBOOST or LGBM API support linear boosting.

linear-tree differs because it's a simple sklearn wrapper that embeds every sklearn.linear_model in the leave... this is the greatest benefit.

In the next months, I plan to make some extensions to the library with other functionalities. If you support the project don't forget to leave a star ;-)
